src/System.Web.OData/System.Web.OData.csproj

```diff
@@ -540,6 +540,7 @@
   </ItemGroup>
   <ItemGroup>
     <None Include="packages.config" />
+    <None Include="Settings.StyleCop" />
```

Comment:
Please revert this change: Settings.StyleCop does not need to be part of the project.